contrib/scripts/env-setup.py

```diff
@@ -39,9 +49,18 @@ def get_py_version():
   return version_tuple[0] + version_tuple[1]  # major_version + minor_version
 
 
+def get_cuda_version():
+  if is_gpu_runtime():
+    return DEFAULT_CUDA_VERSION
+
+  return None  # cuda not available, install non cuda wheels
```

Comment:
In Python usually you do not return anything (which implicitly means None).
Or maybe: return X if Y else Z

Reply:
Done.